### PR TITLE
minor styling

### PR DESCRIPTION
**Pull request by molllusk**

### CODE REVIEW

File: app/assets/stylesheets/application/custom.scss [hunk 1]
```diff
@@ -197,7 +197,19 @@ tr:hover {
 }
 
 #filterrific_filter {
-  width: 50%;
-  margin-left: auto;
-  margin-right: auto;
+  width: 500px;
+  margin-top: 15px;
+
```
Comment (arvel) [inline app/assets/stylesheets/application/custom.scss]: whitespace

File: app/assets/stylesheets/application/custom.scss [hunk 2]
```diff
@@ -197,7 +197,19 @@ tr:hover {
...
+  // margin-left: auto;
+  // margin-right: auto;
+}
+
+.reset_button {
+  margin-top: -40px;
+  margin-left: 400px;
+
```
Comment (arvel) [inline app/assets/stylesheets/application/custom.scss]: whitespace

File: app/views/fluid_inventory_thresholds/index.html.erb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <% provide(:title, 'Fluid Inventory Thresholds') %>
-<h1>Fluid Inventory Thresholds</h1>
+<!-- <h1>Fluid Inventory Thresholds</h1> -->
```
Comment (arvel) [inline app/views/fluid_inventory_thresholds/index.html.erb]: Delete over commenting...

File: app/views/fluid_inventory_updates/_list.html.erb [hunk 4]
```diff
@@ -1,9 +1,9 @@
 <div id="filterrific_results">
 
-  <div class='center' style='margin-bottom: 30px;'>
+  <div class='search_bar' style='margin-bottom: 30px;'>
```
Comment (arvel) [inline app/views/fluid_inventory_updates/_list.html.erb]: if you are styling then you may as well move style='margin-bottom: 30px;' to the css class

Comment (molllusk) [reply app/views/fluid_inventory_updates/_list.html.erb]: i know. i'm lazy.

File: app/views/fluid_inventory_updates/index.html.erb [hunk 5]
```diff
@@ -1,20 +1,20 @@
 <% provide(:title, 'Fluid Inventory Updates') %>
-<h1>Fluid Inventory Updates</h1>
+<!-- <h1>Fluid Inventory Updates</h1> -->
```
Comment (arvel) [inline app/views/fluid_inventory_updates/index.html.erb]: delete

File: app/views/fluid_inventory_updates/index.html.erb [hunk 6]
```diff
@@ -1,20 +1,20 @@
...
 
 <%= form_for_filterrific @filterrific do |f| %>
-  <div class='center' style='margin-bottom: 5px;'>
+  <div class='search_bar' style='margin-bottom: 5px;'>
```
Comment (arvel) [inline app/views/fluid_inventory_updates/index.html.erb]: move my inline styling to class

File: app/views/fluid_inventory_updates/index.html.erb [hunk 7]
```diff
@@ -1,20 +1,20 @@
...
+      class: 'search_field',
       placeholder: ' Search'
     ) %>
   </div>
   <div class='center reset_button' style='margin-bottom: 10px;'>
     <%= link_to(
       'Reset Search',
       reset_filterrific_url,
-      class: "btn btn-sm btn-primary", style: 'color: white') %>
+      class: "reset_search", style: 'color: #337ab7; font-weight: 450;') %>
```
Comment (arvel) [inline app/views/fluid_inventory_updates/index.html.erb]: move styling to class

File: app/views/users/index.html.erb [hunk 8]
```diff
@@ -1,5 +1,5 @@
 <% provide(:title, 'All users') %>
-<h1>All users</h1>
+<!-- <h1>All users</h1> -->
```
Comment (arvel) [inline app/views/users/index.html.erb]: delete

File: app/views/products/index.html.erb [hunk 9]
```diff
@@ -1,20 +1,20 @@
 <% provide(:title, 'All products') %>
-<h1>Search Products</h1>
+<!-- <h1>Search Products</h1> -->
```
Comment (arvel) [inline app/views/products/index.html.erb]: delete

File: app/views/users/index.html.erb [hunk 10]
```diff
@@ -1,5 +1,5 @@
...
 <div class='row'>
```
Comment (arvel) [inline app/views/users/index.html.erb]: Need margin above this

File: app/views/fluid_inventory_thresholds/index.html.erb [hunk 11]
```diff
@@ -1,5 +1,5 @@
...
 
 <%= render(
```
Comment (arvel) [inline app/views/fluid_inventory_thresholds/index.html.erb]: need margin above this

File: app/views/fluid_inventory_updates/_list.html.erb (hunk 4 above)
Comment (molllusk) [reply app/views/fluid_inventory_updates/_list.html.erb]: keeping this one for now.

File: app/views/fluid_inventory_updates/index.html.erb [hunk 12]
```diff
@@ -1,20 +1,19 @@
 <% provide(:title, 'Fluid Inventory Updates') %>
-<h1>Fluid Inventory Updates</h1>
 
 <%= form_for_filterrific @filterrific do |f| %>
-  <div class='center' style='margin-bottom: 5px;'>
+  <div class='search_bar' style='margin-bottom: 5px;'>
     <%# give the search field the 'filterrific-periodically-observed' class for live updates %>
     <%= f.text_field(
       :search_query,
-      class: 'filterrific-periodically-observed',
```
Comment (arvel) [inline app/views/fluid_inventory_updates/index.html.erb]: I think we need this class for the search to work

File: app/views/users/index.html.erb [hunk 13]
```diff
@@ -13,5 +10,10 @@
   </tr>
   <%= render @users %>
 </table>
+<div class='row addauser'>
```
Comment (arvel) [inline app/views/users/index.html.erb]: ew... you are moving this to the bottom?

File: app/assets/stylesheets/application/custom.scss [hunk 14]
```diff
@@ -197,7 +197,30 @@ tr:hover {
...
+}
+
+.reset_search {
+  color: #337ab7;
+  font-weight: 450;
+}
+
+.addauser {
+  margin-top: 10px;
+  float: left;
```
Comment (arvel) [inline app/assets/stylesheets/application/custom.scss]: Don't think you want float: left here... it is pulling it out of the container.

File: app/views/users/index.html.erb [hunk 15]
```diff
@@ -1,8 +1,5 @@
 <% provide(:title, 'All users') %>
-<h1>All users</h1>
-<div class='row'>
-<a href="/users/new" style='float:right' role="button"><button class="btn btn-primary">Add User</button></a>
-</div>
+
 <table class="full_width">
```
Comment (arvel) [inline app/views/users/index.html.erb]: Still need space above this and beneath header (10+ px)... there is some serious visual crowding happening there.

**Pull request merged**